python/lsst/dbb/buffmngrs/handoff/local.py

```diff
@@ -37,14 +38,12 @@
 
 class Finder(Command):
     """Command finding out all files in the buffer on the handoff site.
-
     Parameters
```

Comment:
Please put back the blank line you removed; it's not optional (follow this [link](https://developer.lsst.io/python/numpydoc.html#basic-format-of-docstrings) for details).